template.yaml

```diff
@@ -441,6 +441,29 @@ Resources:
             Method: put
             RestApiId: !Ref ScientificIndexApi
 
+  CreateNotesHandler:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: nvi-rest
+      Handler: no.sikt.nva.nvi.rest.CreateNotesHandler::handleRequest
```

Comment:
Maybe call handler as CreateNoteHandler, since you are creating only one note per request

Reply:
Fixed